content/technotes/2018-05-10-icon.Rmd

```diff
@@ -0,0 +1,57 @@
+---
+slug: "icon"
+title: "icon: web icons for rmarkdown"
+authors:
+  - name: Mitchell O'Hara-Wild
+    url: https://github.com/mitchelloharawild
+date: 2018-05-10
+output: 
+categories:
+- technotes
+topicid: 
+tags:
+- R
+- icon
+- web
+- rmarkdown
+- knitr
+---
+
+```{r setup, include = FALSE}
+knitr::opts_chunk$set(
+  collapse = TRUE,
+  comment = "#>"
+)
+```
+
+# Icons in R <img src="https://i.imgur.com/YTJIhij.png" align="right" />
+The [icon](https://github.com/ropenscilabs/icon) package provides a convenient interface for adding icons from popular web fonts to R Markdown documents. The project began at [rOpenSci OzUnconf 2017](https://ozunconf17.ropensci.org/), and was developed by [Mitchell O'Hara-Wild](https://github.com/mitchelloharawild), [Earo Wang](https://github.com/earowang) and [Timothy Hyndman](https://github.com/timothyhyndman). The package currently supports icons from [Font Awesome](https://fontawesome.com/), [Academicons](http://jpswalsh.github.io/academicons/), and [ionicons](http://ionicons.com/).
+
+Icons can be added to your R Markdown documents using short prefixes which identify the font's library.
+
+* fa: Font Awesome
+* ai: Academicons
+* ii: ionicons
+
+For example, `` `r '\x60r icon::fa("rocket")\x60'` `` can be used to add the [rocket icon](https://fontawesome.com/icons/rocket) from Font Awesome. This interface is convenient if you are familiar with the icon you want, or if you are dynamically selecting your icon.
+
+The package also provides a second interface which allows for tab completion of font names. To include the rocket icon above, you could instead use `` `r '\x60r icon::fa_rocket()\x60'` ``. This is a useful interface for if you are unfamiliar with the available icons or their names.
+
+Each icon function also provides support for modifying the attributes of the icon such as size, colour, rotation, and animation. We can make the rocket icon spin using `` `r '\x60r icon::fa_rocket(animate = "spin")\x60'` ``.
+
+# R in icons
+The long-awaited [R Project icon](http://cran.r-project.org/logo.html) has been added to the Font Awesome library in version 5.0.11. This addition comes nearly four years since the icon's [first request](https://github.com/FortAwesome/Font-Awesome/issues/3862) after being brought into the spotlight of [#rstats](https://twitter.com/hashtag/rstats) with this [tweet](https://twitter.com/LockeData/status/987286302721806337).
```

Comment:
Optional: you could add "with this tweet by [LockeData](https://twitter.com/LockeData)." to give more of a direct shoutout / credit for the signal boost.

Reply:
Is it possible to embed tweets in posts?
https://dev.twitter.com/web/embedded-tweets

Reply:
Yes! embedding would be great